projects/packages/sync/src/class-functions.php

```diff
@@ -22,7 +22,9 @@ class Functions {
 	 * @return array
 	 */
 	public static function get_modules() {
-		require_once JETPACK__PLUGIN_DIR . 'class.jetpack-admin.php';
+		if ( defined( 'JETPACK__PLUGIN_DIR' ) ) {
```

Comment:
class.jetpack-admin.php defines the Jetapck_Admin class.

This change will lead to fatals as Jetpack_Admin will not be defined if the Sync package is being used without the full plugin.

I suggest that get_modules return an empty array in this case. Perhaps have the existing line 29 move up below the require statement and replace line 29 with return [];

Reply:
Fixed.